RNA-seq/07-bulk_rnaseq_exercise.Rmd

```diff
@@ -0,0 +1,160 @@
+
+These data were published as part of Sachs et al. _Blood._ 2014.
+
+### Setup 
+
+We've prepared the expression data all the way through the tximport step.
+This file can be found at: `data/tximport/leukemia/leukemia_stem_cell_txi.RDS`
+
+The metadata for this experiment can be found in this TSV file: `data/sample_metadata/leukemia/SRP049821_metadata.tsv`
+
+Using what we've learned so far, let's ask a series of questions about these
+data using DESeq2.
+
+```{r}
+library(DESeq2)
+```
+
+We'll use the `%>%` operator as well.
+
+```{r}
+`%>%` <- dplyr::`%>%`
+```
+
+### What does the relationship between the cell populations look like?
+
+Read in the gene expression data and the metadata.
+The cell population information is in a column called `cell_sorting` in the
+metadata file.
+
+```{r}
+# call the tximport processed data txi
+txi <- 
+```
+
+```{r}
+# call the metadata data.frame metadata_df
+metadata_df <- 
+```
+
+Use DESeq2 functionality to create a dataset, transform, and visualize the data.
+Recall that we are interested in the cell populations.
+
+```{r}
+
+```
+
+```{r}
+
+```
+
+```{r}
+
+```
+
+From Sachs et al. _Blood._ 2014 (read it [here](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4239336/)):
+
+> ...these data identify the Mac-1LowKit+Sca-1+ group as enriched with [leukemia stem-cells] compared with the other groups, particularly the Mac1High group that accounts for the majority of the leukemia population.
+...
+Both of these modalities identify the Mac-1High group as the group with the least stem-cell capacity.
+
+Do the gene expression profiles align with the results from the Sachs et al. 
+experiments?
+
+### What genes are highly expressed in cells with high stem-cell capacity?
+
+Let's compare the `MacLowKitposSca1pos` (high capacity) to `MacHigh` (low 
+capacity) cells.
+First, we'll filter the tximport-processed data to only include samples
+from these two groups.
+
+```{r}
+str(txi)
```

Comment:
Could ask them to use `str` here rather than providing it to them. I think we have them use `str` enough that they *should* remember how to use it. Plus its not a super complicated function to use. Good to build up confidence early in this notebook.